lib/routes/defaults.js

```diff
@@ -27,33 +29,66 @@ module.exports = function (log, P, db, error) {
   function versionHandler(request, reply) {
     log.begin('Defaults.root', request)
 
+    // Get the version from fxa-auth-db-mysql server
+    function getDBVersion() {
+      return new P(function (resolve, reject) {
+        if (!dbVersion) {
+          var config = require('../../config').root()
+          httprequest.get({url: config.httpdb.url, json: true, timeout: 10000}, function (err, response, body) {
```

Comment:
@jrgm there is a 10 sec timeout here, so we can give it a try? (maybe with a shorter timeout) to see how well it works in stage and prod